Jgersztyn/SimpleQueueApplication
# Clean up Structure

What does this pull request do?

- Changes the structure of the service to simplify object creation
- Cleaned up code and comments

How should it be tested?
- Add a connection string to Azure Service Bus and a queue name to the connection information. Run the application. The service should successfully post a message to the queue.
